### DISCUSSION TITLE
State of Mojo docs

### DISCUSSION THREAD

**Discussion by gryznar** [modular/modular#441] [Q&A]
I am wondering in which state currently are docs? In Python docs are explaining concepts very well. On the other hand in Mojo it seems, that they only "touch the surface". Mojo has great low level features which may be very helpful in optimizing code, but they are not explained so well in my opinion.
Is it going to change in the future (with examples of usages, better definition etc)?

To paraphrase: are they going to be more begginer friendly? Currently there is a need to use external source of knowledge to familiarize with some advanced concepts. A lot of these explaination may be put directly into docs.

I am aware that Mojo is under extensive and rapid development, so docs should have lower priority. I am just asking if there are plans to improve current state in the nearest or farest?

**Comment by scottamain** [comment]
Absolutely. Please keep in mind that the Mojo language came into existence only [in September](https://docs.modular.com/mojo/changelog.html). There is still a lot that has not been documented, and a lot that [still does not exist](https://docs.modular.com/mojo/roadmap.html).

We have big plans for the Mojo language, and the documentation is no exception. But this is a years-long journey we've embarked upon. Stay tuned for both step-wise and continuous improvements to the docs. :)

**Reply by gryznar** [reply]
I was almost sure, that's the temporary look :) Nice to see the confirmation!
